libnd4j/include/helpers/TAD.h

```diff
@@ -727,23 +720,23 @@ SD_INLINE sd::LongType *TAD::shapeInfoOnlyShapeAndStride() {
     }
   }
 
-  delete[] permuted;
-  delete[] newPermuteDims;
-  delete[] rankRange;
-  delete[] remove;
-  delete[] reverseDimensions;
+ // delete[] permuted;
+ // delete[] newPermuteDims;
+ // delete[] rankRange;
+//  delete[] remove;
+//  delete[] reverseDimensions;
```

Comment:
this looks suspicious, are you certain you don't need to free this memory?

Reply:
done